l1-contracts/contracts/state-transition/chain-deps/facets/Admin.sol

```diff
@@ -115,8 +125,11 @@ contract AdminFacet is ZkSyncHyperchainBase, IAdmin {
     }
 
     /// @inheritdoc IAdmin
-    function setPubdataPricingMode(PubdataPricingMode _pricingMode) external onlyAdmin onlyL1 {
-        require(s.totalBatchesCommitted == 0, "AdminFacet: set validium only after genesis"); // Validium mode can be set only before the first batch is processed
+    function setPubdataPricingMode(PubdataPricingMode _pricingMode) external onlyAdmin {
```

Comment:
onlyL1 was removed, do we want that? It is ok for setTokenMultiplier

Reply:
good call!